src/tests/test_converter.py

```diff
@@ -22,3 +23,92 @@ def test_apply_filters(self):
         ]
         result = _apply_filters("a", filters)
         self.assertEqual(result, "A")
+
+    def test_is_object_reference(self):
+        testcases = (
+            ("this.andthat", True),
+            ("JKDldjkd.jklfD", True),
+            (["ajkdlf", ".", "jdkladf"], False),
+            ("kdkdkdfklajdf", False),
+            (".", False),
+            ("kdkakdkaf.", False),
+            (".kdljfakld", False),
+            ("", False),
+            ("kdjld_.dkdj_dkldjf", True),
+        )
+
+        for testcase, result in testcases:
+            self.assertEqual(result, _is_object_reference(testcase), "Case {} should return {}".format(testcase, result))
+
```

Comment:
I find
f"Case {testcase} should return {result}"
more readable

Reply:
You're right, if only for consistency :) Changed